rocky/rocky/templates/crisis_room/crisis_room_findings_block.html

```diff
@@ -8,129 +8,137 @@ <h1>{% translate 'Findings per organization' %}</h1>
       {% else %}
         <h1>{% translate 'Total findings' %}</h1>
       {% endif %}
-      <table>
-        <thead>
-          <tr>
-            <th>{% translate 'Organization' %}</th>
-            <th scope="col" class="sortable" abbr="Sortable data">{% translate 'Total Findings' %}</th>
-            <th scope="col">{% translate 'Details' %}</th>
-          </tr>
-        </thead>
-        <tbody>
-          {% for findings in findings_per_org_total %}
+      {% if organizations %}
```

Comment:
I guess the main change here is this if-wrappping?

Reply:
Yes, it is. Because being suspended can lock you out of one or more organisations, the list could be empty. And rendering an empty table without an informational message seemed dull :)